frontend/src/components/RejectionReason/Reasons/TokenModule.vue

```diff
@@ -0,0 +1,112 @@
+<template>
+	<span>
+		<!-- Transaction is rejected because of Insufficent balance -->
+		<span v-if="reason.eventType == 'tokenBalanceInsufficient'">
+			<Tooltip :text="reason.eventType" text-class="text-theme-body">
+				<span class="px-2">
+					Token holder transaction rejected because due to insufficient balance.
+				</span>
+				<br />
+				<span class="px-2">
+					Required balance:
+					<b>
+						{{
+							(
+								Number(
+									reason.details.requiredBalance[1][
+										'$serde_json::private::Number'
```

Comment:
I would prefer the frontend did not rely on internal details of the backend, like the fact we are using `serde_json`.
I don't think we should even leak that information in the API.

Reply:
I don't want this to be a blocking issue, if you can find a fix great, but otherwise I would be fine with merging it in and we can address it later.
I expect we will iterate on the content of these anyway, so we will probably get back to this again.